bitcoind-rpc/src/main/scala/org/bitcoins/rpc/serializers/JsonSerializers.scala

```diff
@@ -1,386 +1,410 @@
+      (__ \ "time").read[UInt32] and
+      (__ \ "timereceived").readNullable[UInt32] and
+      (__ \ "comment").readNullable[String] and
+      (__ \ "to").readNullable[String] and
+      (__ \ "otheraccount").readNullable[String] and
+      (__ \ "bip125-replaceable").read[String] and
+      (__ \ "abandoned").readNullable[Boolean])(ListTransactionsResult)
+
+  implicit val unspentOutputReads: Reads[UnspentOutput] =
+    Json.reads[UnspentOutput]
+
+  // Other Models
+  implicit val blockTransactionReads: Reads[BlockTransaction] =
+    Json.reads[BlockTransaction]
+  implicit val getBlockTemplateResultReads: Reads[GetBlockTemplateResult] =
+    Json.reads[GetBlockTemplateResult]
+
+  implicit val miningInfoReads: Reads[GetMiningInfoResult] =
+    Json.reads[GetMiningInfoResult]
+
+  implicit val memoryManagerReads: Reads[MemoryManager] =
+    Json.reads[MemoryManager]
+  implicit val getMemoryInfoResultReads: Reads[GetMemoryInfoResult] =
+    Json.reads[GetMemoryInfoResult]
+
+  implicit val validateAddressResultReads: Reads[ValidateAddressResultImpl] =
+    Json.reads[ValidateAddressResultImpl]
+
+  implicit val embeddedResultReads: Reads[EmbeddedResult] =
+    Json.reads[EmbeddedResult]
+
+  implicit val addressInfoResultReads: Reads[AddressInfoResult] =
+    Json.reads[AddressInfoResult]
+
+  implicit val receivedLabelReads: Reads[ReceivedLabel] =
+    Json.reads[ReceivedLabel]
+
+  implicit val estimateSmartFeeResultReads: Reads[EstimateSmartFeeResult] =
+    Json.reads[EstimateSmartFeeResult]
+
+  implicit val walletProcessPsbtResultReads: Reads[WalletProcessPsbtResult] =
+    Json.reads[WalletProcessPsbtResult]
+
+  implicit val finalizedPsbtReads: Reads[FinalizedPsbt] = FinalizedPsbtReads
+
+  implicit val nonFinalizedPsbtReads: Reads[NonFinalizedPsbt] =
+    NonFinalizedPsbtReads
+
+  implicit val finalizePsbtResultReads: Reads[FinalizePsbtResult] =
+    FinalizePsbtResultReads
+
+  implicit val rpcPsbtOutputReads: Reads[RpcPsbtOutput] = RpcPsbtOutputReads
+
+  implicit val psbtBIP32DerivsReads: Reads[PsbtBIP32Deriv] =
+    PsbtBIP32DerivsReads
+
+  implicit val rpcPsbtScriptReads: Reads[RpcPsbtScript] = RpcPsbtScriptReads
+
+  implicit val psbtWitnessUtxoInputReads: Reads[PsbtWitnessUtxoInput] =
+    Json.reads[PsbtWitnessUtxoInput]
+
+  implicit val mapPubKeySignatureReads: Reads[
+    Map[ECPublicKey, ECDigitalSignature]] = MapPubKeySignatureReads
+
+  implicit val rpcPsbtInputReads: Reads[RpcPsbtInput] = RpcPsbtInputReads
+
+  implicit val decodePsbtResultReads: Reads[DecodePsbtResult] =
+    Json.reads[DecodePsbtResult]
+
+  implicit val psbtMissingDataReads: Reads[PsbtMissingData] =
+    Json.reads[PsbtMissingData]
+
+  implicit val analyzePsbtInputReads: Reads[AnalyzePsbtInput] =
+    Json.reads[AnalyzePsbtInput]
+
+  implicit val analyzePsbtResultReads: Reads[AnalyzePsbtResult] =
+    Json.reads[AnalyzePsbtResult]
+
+  implicit val getNodeAddressesReads: Reads[GetNodeAddressesResult] =

```

Comment:
When pulling down this I get an error about not having a reader for `FiniteDuration`. We could solve this by implementing a generic reader for `FiniteDuration`s, but I'd say that's not a very good idea. The reason for that is that `FiniteDuration`s can be different time units (seconds, millis, hours, etc). The generic JSON reader wouldn't know which time unit we're dealing with. So the correct solution here is to be a bit more verbose, and implement the reader manually:

```scala
implicit val getNodeAddressesReads: Reads[GetNodeAddressesResult] = { js =>
    for {
      time <- js.validate[Long].map(FiniteDuration(_, TimeUnit.SECONDS))
      services <- js.validate[Int]
      address <- js.validate[URI]
      port <- js.validate[Int]
    } yield GetNodeAddressesResult(time, services, address, port)
  }
```